lib/galaxy/webapps/galaxy/fast_app.py

```diff
@@ -72,6 +77,22 @@ async def add_x_frame_options(request: Request, call_next):
             response.headers['X-Frame-Options'] = x_frame_options
             return response
 
+    nginx_x_accel_redirect_base = getattr(gx_app.config, 'nginx_x_accel_redirect_base', None)
```

Comment:
```suggestion
    nginx_x_accel_redirect_base = gx_app.config.nginx_x_accel_redirect_base
```
It's in the schema, so it's an attribute of `gx_app`. I've checked how `app` is created in fastapi_factory - I assume the config object is always initialized at this point (I don't think there's an edge case where the config is empty, or is there?)

Reply:
This was just a *monkey see monkey do* from the `x_frame_options` above :sweat_smile: I guess that one will also be safe to directly get the value from `gx_app.config.x_frame_options` then.

I will change those too, Thanks @ic4f!

Reply:
Thanks!  :rofl: :rofl: